ROMUtils.cpp

```diff
@@ -146,6 +146,45 @@ namespace ROMUtils
         return ret;
     }
 
+    /// <summary>
+    /// Get an 32 bytes uchar array of Tile8x8 graphic data, then X flip the data of its graphic.
+    /// </summary>
+    /// <param name="source">
+    /// The pointer to read uchar array Tile8x8 data.
+    /// </param>
+    /// <param name="destination">
+    /// The pointer to save changed Tile8x8 data to.
+    /// </param>
+    void Tile8x8DataXFlip(unsigned char *source, unsigned char *destination)
```

Comment:
Is there perhaps a scenario where we want to read the tile with no flip? Or perhaps X and Y flip? This seems like something that should be one function with bool xFlip, bool yFlip. What do you think about this?